hologram_dashboard/fetcher/BUILD

```diff
@@ -6,19 +6,30 @@ cc_library(
 )
 
 cc_library(
-    name = "hologram_data_source_availabiliity_fetcher",
-    srcs = ["hologram_data_source_availabiliity_fetcher.cc"],
+    name = "hologram_data_source_availability_fetcher",
+    srcs = ["hologram_data_source_availability_fetcher.cc"],
```

Comment:
super nit: I feel this class name is too long, and no need to include hologram. Can we name it like data_availability_fetcher?